tasks/cephfs/cephfs_test_case.py

```diff
@@ -15,11 +22,55 @@ class CephFSTestCase(unittest.TestCase):
     Handles resetting the cluster under test between tests.
     """
     # Environment references
+    mounts = None
+    fs = None
+    ctx = None
+
+    # FIXME weird explicit naming
```

Comment:
> Going to fix this?

Probably at some point -- but this is a pre-existing thing, it just used to be duplicated through the cases and now it's done in this test case.

The thing I'm remind myself about here is that it's weird to have mount_a and mount_b instead of just using mounts[0] and mounts[1]